src/jupyterlabcontroller/models/v1/lab.py

```diff
@@ -31,9 +31,12 @@
 
 
 class LabSize(str, Enum):
-    """Allowable names for pod sizes."""
+    """Allowable names for pod sizes.
```

Comment:
I prefer quoting the actual SRD document rather than a derivative work, even if the derivative work does have a much more descriptive URL.

Reply:
Well, more to the point it's a much shorter URL so I can move it into a docstring without flake8 whining at me (otherwise, yes, I'm on board).

Reply:
I moved the link to the rST epilogue which avoids the line length issue.